TOMP-API.yaml

```diff
@@ -1,12 +1,13 @@
 openapi: 3.0.0
 info:
   title: Transport Operator MaaS Provider API
-  description: An API between MaaS providers and transport operators for booking trips and corresponding assets.
+  description:
+    An API between MaaS providers and transport operators for booking trips and corresponding assets.
     <p>The documentation (examples, process flows and sequence diagrams) can be found at <a href="https://github.com/TOMP-WG/TOMP-API/">github</a>.
-  version: "0.5.2"
+  version: "0.5.3"
```

Comment:
Is there a particular reason you think we should switch to double quotes everywhere? I don't have a preference but we were already quite consistent in using single quotes so far.

Reply:
I don't have a strong preference either and will defer to your judgement. I just tend to not fight the linter. 

The YAML validation settings seem to come from the [Red Hat YAML language support](https://github.com/redhat-developer/vscode-yaml) VSCode plugin.